util_inference_example.py

```diff
@@ -0,0 +1,87 @@
+"""
+Usage: python util_inference_example.py image annotations
+
+Adapted from the dense_inference.py to demonstate the usage of the util
+functions.
+"""
+
+import sys
+import numpy as np
+import cv2
+import densecrf as dcrf
+import matplotlib.pylab as plt
+from skimage.segmentation import relabel_sequential
+
+from utils import compute_unary, create_pairwise_bilateral, \
+    create_pairwise_gaussian
```

Comment:
Yes I agree, I think densecrf should be a python package. In normal python we should have than the `DenseCRF` class (and `DenseCRF2D`) in the `__init__.py`. Then `utils.py` should be a separate file in the package. This would then allow to do 
```
from densecrf import DenseCRF
from densecrf.utils import compute_unary
```
which I think is the desired behavior of the library. I'm not an expert with cython, so I'm not sure if it is possible to put cython code in the `__init.py__` (I guess the code from `densecrf.pyc` would be what we would like to have in their).